# This is a message for storing a 2D object detection and corresponding 3D point description
graspberry_perception/BoundingBox bounding_box
graspberry_perception/ObjectDescription description
